pkg/dashboards/types.go

```diff
@@ -439,15 +460,17 @@ type DashboardVariableInput struct {
 	// [DEPRECATED] Default value for this variable. The actual value to be used will depend on the type.
 	DefaultValue *DashboardVariableDefaultValueInput `json:"defaultValue,omitempty"`
 	// Default values for this variable. The actual value to be used will depend on the type.
-	DefaultValues *[]DashboardVariableDefaultItemInput `json:"defaultValues,omitempty"`
+	DefaultValues []*DashboardVariableDefaultItemInput `json:"defaultValues,omitempty"`
 	// Indicates whether this variable supports multiple selection or not. Only applies to variables of type NRQL or ENUM.
 	IsMultiSelection bool `json:"isMultiSelection,omitempty"`
 	// List of possible values for variables of type ENUM
 	Items []DashboardVariableEnumItemInput `json:"items,omitempty"`
 	// Configuration for variables of type NRQL.
-	NRQLQuery *DashboardVariableNRQLQueryInput `json:"nrqlQuery,omitempty"`
+	NRQLQuery DashboardVariableNRQLQueryInput `json:"nrqlQuery,omitempty"`
 	// Variable identifier.
 	Name string `json:"name"`
+	// Options applied to the variable
+	Options DashboardVariableOptionsInput `json:"options,omitempty"`
 	// Indicates the strategy to apply when replacing a variable in a NRQL query.
 	ReplacementStrategy DashboardVariableReplacementStrategy `json:"replacementStrategy,omitempty"`
```

Comment:
```go
// DashboardVariableInput - Definition of a variable that is local to this dashboard. Variables are placeholders for dynamic values in widget NRQLs.
type DashboardVariableInput struct {
	// [DEPRECATED] Default value for this variable. The actual value to be used will depend on the type.
	DefaultValue *DashboardVariableDefaultValueInput `json:"defaultValue,omitempty"`
	// Default values for this variable. The actual value to be used will depend on the type.
	DefaultValues *[]DashboardVariableDefaultItemInput `json:"defaultValues,omitempty"`
	// Indicates whether this variable supports multiple selection or not. Only applies to variables of type NRQL or ENUM.
	IsMultiSelection bool `json:"isMultiSelection,omitempty"`
	// List of possible values for variables of type ENUM
	Items []DashboardVariableEnumItemInput `json:"items,omitempty"`
	// Configuration for variables of type NRQL.
	NRQLQuery *DashboardVariableNRQLQueryInput `json:"nrqlQuery,omitempty"`
	// Variable identifier.
	Name string `json:"name"`
	// Options applied to the variable
	Options DashboardVariableOptionsInput `json:"options,omitempty"`
	// Indicates the strategy to apply when replacing a variable in a NRQL query.
	ReplacementStrategy DashboardVariableReplacementStrategy `json:"replacementStrategy,omitempty"`
	// Human-friendly display string for this variable.
	Title string `json:"title,omitempty"`
	// Specifies the data type of the variable and where its possible values may come from.
	Type DashboardVariableType `json:"type"`
}
```

Reply:
Upon seeing all the above changes I realised there is a gap between tutone generation and manual modification. I will check if these manual changes need to be accommodated